apps/wallet-mobile/src/i18n/LanguageProvider.tsx

```diff
@@ -113,7 +122,7 @@ const useSaveLanguageCode = ({onSuccess, ...options}: UseMutationOptions<void, E
   const mutation = useMutation({
     mutationFn: (languageCode) => storage.join('appSettings/').setItem('languageCode', languageCode),
     onSuccess: (data, languageCode, context) => {
-      updateLanguageSettings(defaultLanguageCode)
+      updateLanguageSettings(languageCode)
```

Comment:
this was sounding odd, why was it updating with `defaultLanguageCode` ? the update will be affected only after quitting and reopening the app. Dunno if this was the desired behaviour.

Reply:
Removed, doesn't do anything anyway